### PR TITLE
create fake api

### PR DESCRIPTION
**Pull request by mbob72**

### CODE REVIEW

**Comment by Michael-Ananev** [comment]
Added fake Api

File: src/todo/components/TodoList.tsx [hunk 1]
```diff
@@ -36,6 +38,7 @@ const ToDo = () => {
 
     return (
         <>
+            <button onClick={() => getList().then(console.log).catch(console.error)}>Try api</button>
```
Comment (dmitrymokienko) [inline src/todo/components/TodoList.tsx:41]: [optional] better move to the handler func

**Review by dmitrymokienko — APPROVED** [review]

File: src/todo/components/TodoList.tsx (hunk 1 above)
Comment (Michael-Ananev) [reply src/todo/components/TodoList.tsx:41]: I think it's already fixed

**Comment by Michael-Ananev** [comment]
I think this is OK

**Pull request closed without merging**